src/main/java/com/epam/training/olha_haichenkova/task_3/page/PricingCalculatorPage.java

```diff
@@ -0,0 +1,198 @@
+            vMFamilyOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectVMSeries(){
+        vMSeriesSelectIcon.click();
+        if(!vMSeriesOption.isSelected()){
+            vMSeriesOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectMachineType(){
+        machineTypeSelectIcon.click();
+        wait.until(ExpectedConditions.elementToBeClickable(machineTypeOption));
+        if(!machineTypeOption.isSelected()){
+            machineTypeOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectGPU(){
+        addGPUsCheckbox.click();
+        gPUTypeSelectIcon.click();
+        if(!gPUTypeOption.isSelected()){
+            gPUTypeOption.click();
+        }
+        numberOfGPUsSelectIcon.click();
+        wait.until(ExpectedConditions.elementToBeClickable(numberOfGPUsOption));
+        if(!numberOfGPUsOption.isSelected()){
+            numberOfGPUsOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectLocalSSD(){
+        localSSDSelectIcon.click();
+        wait.until(ExpectedConditions.elementToBeClickable(localSSDOption));
+        if(!localSSDOption.isSelected()){
+            localSSDOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectDatacenterLocation() {
+        datacenterLocationSelectIcon.click();
+        wait.until(ExpectedConditions.elementToBeClickable(datacenterLocationOption));
+        if (!datacenterLocationOption.isSelected()) {
+            datacenterLocationOption.click();
+        }
+        return this;
+    }
+
+    private PricingCalculatorPage selectCommittedUsage() {
+        committedUsageSelectIcon.click();
+        wait.until(ExpectedConditions.elementToBeClickable(committedUsageOption));
+        if (!committedUsageOption.isSelected()) {
+            committedUsageOption.click();
+        }
+        return this;
+    }
+
+    public CalculationResultsPage fillInCalculationForm(int numberOfVM) {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By
+                        .xpath(OUTER_IFRAME_XPATH)));
+        WebElement outerIframe = driver.findElement(By.xpath(OUTER_IFRAME_XPATH));
+        driver.switchTo().frame(outerIframe);
+        driver.switchTo().frame(INNER_IFRAME_ID);
+
+        checkIfComputeEngineSelected()
+                .selectNumberOfServers(numberOfVM)
+                .selectOperatingSystemSoftware()
+                .selectVMClass()
+                .selectVMSeries()
+                .selectMachineType()
+                .selectGPU()
+                .selectLocalSSD()
+                .selectDatacenterLocation()
+                .selectCommittedUsage();
+
+        addToEstimateButton.click();
+        wait.until(ExpectedConditions.not(ExpectedConditions.urlToBe(PRICING_CALCULATOR_URL)));
+        return new CalculationResultsPage(driver);
+    }
```

Comment:
We should be able to choose other options. Because if we'll need to change just one parameter, we have to duplicate this method, it's not flexible